zerver/webhooks/intercom/view.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+from typing import Any, Dict, Iterable, Text
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@api_key_only_webhook_view('Intercom')
+@has_request_variables
+def api_intercom_webhook(request: HttpRequest, user_profile: UserProfile,
+                         payload: Dict[str, Any] = REQ(argument_type='body'),
+                         stream: Text = REQ(default='test')) -> HttpResponse:
+    topic = str(payload['topic'])
+    topic = topic.replace('.', ' ').capitalize()
```

Comment:
Should this be capitalized?

Reply:
If its not, it will be `user tag created` instead of 'User tag created'. 'User' is the first word in the reply body.

Reply:
I don't think it should be capitalized, I would suggest leave it as it is! :)